py/config.py

```diff
@@ -1,21 +1,21 @@
 ENTITY_PREFIX = "e-"
 PREDICATE_PREFIX = "p-"
 
-EPOCHS = 3
+EPOCHS = 2                      #changed for BERT
 BATCH_SIZE = 16
 VAL_BATCH_SIZE = 1
 VAL_SHUFFLE = False
 MAX_LAYERS = 2
 MAX_ENTITY_TOKENS = 5
-LEARNING_RATE = 0.1
+LEARNING_RATE = 0.01                #changed for BERT
 
 CELL_STATE_CLAMP_VAL = 1e4
 HIDDEN_STATE_CLAMP_VAL = 1e6
 
-PREPPED_DATA_PATH = "../data/prepped_dataset_7.pickle"
+PREPPED_DATA_PATH = "../data/prepped_dataset.pickle"
 XML_PATH = "../data/BioInfer_corpus_1.1.1.xml"
 
-WORD_EMBEDDING_DIM = 256
+WORD_EMBEDDING_DIM = 768                        #changed for BERT
```

Comment:
we shouldn't need to edit this for BERT... this should only get used when the encoding_method is from-scratch.